### PR TITLE
Provides the initial work on the new standard UI "Tycho"

### PR DESCRIPTION
**Pull request by andyHa**
Also all standard templates of sirius-web are ported to tycho.

Note that this is a non-breaking change as all legacy templates
are still accessible in the "w" namespace. New templates
are in "t" (as in Tycho").

grafik

grafik

grafik

grafik

### CODE REVIEW

File: src/main/resources/default/templates/system/state.html.pasta [hunk 1]
```diff
@@ -3,145 +3,101 @@
...
                 </i:if>
             </i:for>
-        </table>
 
-    </i:for>
-</w:page>
+            <i:block name="footer">
+                <i:switch test="@cluster.getNodeState().name()">
+                    <i:block name="GRAY">
+                        <t:dot color="green">Norminal</t:dot>
```
Comment (qw3ry) [inline src/main/resources/default/templates/system/state.html.pasta:59]:
```suggestion
                        <t:dot color="green">Normal</t:dot>
```

File: src/main/resources/default/templates/system/state.html.pasta [hunk 2]
```diff
@@ -3,145 +3,101 @@
...
+                    </i:block>
+                    <i:block name="GREEN">
+                        <t:dot color="green">Norminal</t:dot>
```
Comment (qw3ry) [inline src/main/resources/default/templates/system/state.html.pasta:62]: same

File: src/main/resources/default/templates/system/state.html.pasta [hunk 3]
```diff
@@ -3,145 +3,101 @@
...
+                    </i:if>
+                </i:for>
+
+                <i:block name="footer">
+                    <i:switch test="@cluster.getNodeState().name()">
+                        <i:block name="GRAY">
+                            <t:dot color="green">Norminal</t:dot>
+                        </i:block>
+                        <i:block name="GREEN">
+                            <t:dot color="green">Norminal</t:dot>
```
Comment (qw3ry) [inline src/main/resources/default/templates/system/state.html.pasta:90]: .

File: src/main/resources/default/assets/tycho/styles/buttons.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.btn-outline-link {
```
Comment (mkeckmkeck) [inline src/main/resources/default/assets/tycho/styles/buttons.scss:1]: Add copyright header

File: src/main/resources/default/assets/tycho/styles/misc.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.cursor-pointer {
```
Comment (mkeckmkeck) [inline src/main/resources/default/assets/tycho/styles/misc.scss:2]: Add copyright header

File: src/main/resources/default/assets/tycho/styles/page.scss [hunk 6]
```diff
@@ -1,5 +1,5 @@
-#wrapper-body {
-  background-color: #FAFAFA;
+body {
```
Comment (mkeckmkeck) [inline src/main/resources/default/assets/tycho/styles/page.scss:1]: Add copyright header

File: src/main/resources/default/assets/tycho/styles/sidebar.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
+.nav-header {
```
Comment (mkeckmkeck) [inline src/main/resources/default/assets/tycho/styles/sidebar.scss:1]: Add copyright header

File: src/main/java/sirius/web/controller/UserMessage.java [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.controller;
+
+import sirius.kernel.commons.Strings;
+import sirius.web.templates.ContentHelper;
+
+public class UserMessage {
```
Comment (mkeckmkeck) [inline src/main/java/sirius/web/controller/UserMessage.java:14]: Javadoc

File: src/main/java/sirius/web/controller/UserMessage.java [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+    private MessageLevel type;
+    private String html;
+
+    public static class Builder {
```
Comment (mkeckmkeck) [inline src/main/java/sirius/web/controller/UserMessage.java:19]: javadoc

File: src/main/java/sirius/web/controller/UserMessage.java [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+        public UserMessage build() {
+            if (Strings.isEmpty(html)) {
+                return null;
+            } else {
+                return new UserMessage(type, html);
+            }
+        }
+    }
+
+    public UserMessage(MessageLevel type, String html) {
```
Comment (mkeckmkeck) [inline src/main/java/sirius/web/controller/UserMessage.java:52]: Is this public on purpose?

**Review by mkeckmkeck — CHANGES_REQUESTED** [review]

File: src/main/resources/default/templates/system/state.html.pasta (hunk 1 above)
Comment (andyHa) [reply src/main/resources/default/templates/system/state.html.pasta:59]: Aber spaceX und NASA dürfen dass au immer so schreiben, wenn se Raketen starten :(

Comment (qw3ry) [reply src/main/resources/default/templates/system/state.html.pasta:59]: In dem Fall drücke ich ein Auge zu

**Review by qw3ry — APPROVED** [review]

**Pull request merged**